src/ensembl/src/shared/components/modal/Modal.scss

```diff
@@ -21,6 +21,12 @@
   box-shadow: 0px 3px 6px #0000004D; // TODO: resolve box-shadow colour globally w/Andrea
 }
 
+@media (max-width: 980px) {
```

Comment:
Why do we have `980` here? I see the horizontal scroll bar appearing until `1116px`. I wonder if this value should be a little higher?